File: Web/Direct4Me.Minimal.Api/Models/Postbox.cs
using System.ComponentModel.DataAnnotations;

namespace Direct4Me.Minimal.Api.Models;

[Serializable]
public record Postbox(
    string Guid,
    [Required] int PostboxId,
    [Required] string UserGuid,
    int TotalUnlocks,
    DateTime LastAccess
);

File: Mobile/app/src/main/java/com/um/feri/direct4me/android/model/Postbox.kt
package com.um.feri.direct4me.android.model

import jakarta.validation.constraints.NotNull
import java.util.*

data class Postbox(
    @field:NotNull val guid: String,
    @field:NotNull val postboxId: Int,
    @field:NotNull val userGuid: String,
    val totalUnlocks: Int,
    @field:NotNull val lastAccess: Date
)